.kokoro/build.sh

```diff
@@ -38,23 +39,41 @@ fi
 
 case ${JOB_TYPE} in
 test)
-    mvn test -B -Dclirr.skip=true
+    mvn test -B -Dclirr.skip=true -Denforcer.skip=true
     bash ${KOKORO_GFILE_DIR}/codecov.sh
     bash .kokoro/coerce_logs.sh
     ;;
 lint)
-    mvn com.coveo:fmt-maven-plugin:check
+    mvn \
+      -Penable-samples \
```

Comment:
This might break the build, I can't find any profile named enable-samples

Reply:
These won't run until we set up a build config and move the samples into this repo, so should be ok. Presubmits are all still passing with this change